### ISSUE TITLE
Issue with "ARIA: listbox role": broken link Option Role"

### ISSUE THREAD

**Issue by corskester** [mdn/content#1938]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

#### What information was incorrect, unhelpful, or incomplete?
there is a link under "See also" that refers to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Option_role as "Aria: Option role", however, this is a non-existent page.

#### Specific section or headline?
See also

#### What did you expect to see?
A page that specifies more information about role="option"

#### Did you test this? If so, how?
Clicking the link and visiting the (Not found) page

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/listbox_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/accessibility/aria/roles/listbox_role/index.html
* Last commit: https://github.com/mdn/content/commit/9b5cf04786dad342498374e51aacb2ccd1999aac
* Document last modified: 2021-01-25T01:09:48.000Z

</details>

**Comment by chrisdavidmills** [comment]
The option role page doesn't exist yet, so to fix this properly, the assignee would have to create a new page for it.

**Comment by chrisdavidmills** [comment]
To whoever fixes this; you would need to:

1. Create a new page in the appropriate place to cover the `option` role; use one of the existing role pages as a template.
2. Write the new content, researching what it should be if necessary.
3. Submit a PR to add the new page to the site.

**Comment by zacnomore** [comment]
I'm going to take a crack at this

**Comment by zacnomore** [comment]
@chrisdavidmills I created the summary and description sections this morning and drafted this PR: https://github.com/mdn/content/pull/9019
Could I get some guidance to make sure this is headed in the right direction?

**Comment by hamishwillee** [comment]
@zacnomore Chris has move to another job. Looks like you have  a reviewer on the PR already though.
